src/api-reference/direct-connects/hotel-service-2/Availability.markdown

```diff
@@ -79,6 +80,10 @@ Message to retrieved the availability of hotels.
           <HotelSearchCriteria>
             <Criterion>
               <HotelRef ChainCode="ZZ" HotelCode="111222"></HotelRef>
```

Comment:
Right now, we send one hotelRef per Criterion. Doing this will mean we need to repeat ratePlanCandidate within every single Criterion.

From schema, it does seem we should be able to send multiple HotelRef within a single criterion object. This will be better as no repeat of RatePlanCandidate. But not sure if current vendor  supports it

Reply:
I will do a test locally and see

Reply:
tested, the big four vendors only expect 1 hotelRef inside each Criteria. So either vendors need to change, or we pass in RatePlanType inside each Criterion